components/host-dashboard/reports-section/TotalMoneyManagedSection.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/client';
+import { Dollar } from '@styled-icons/boxicons-regular/Dollar';
+import { InfoCircle } from '@styled-icons/boxicons-regular/InfoCircle';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { formatCurrency } from '../../../lib/currency-utils';
+import { API_V2_CONTEXT, gqlV2 } from '../../../lib/graphql/helpers';
+
+import Container from '../../Container';
+import { Flex } from '../../Grid';
+import Loading from '../../Loading';
+import StyledCard from '../../StyledCard';
+import StyledTooltip from '../../StyledTooltip';
+import { P, Span } from '../../Text';
+
+const metricsQuery = gqlV2/* GraphQL */ `
+  query HostMetricsQuery($slug: String) {
+    host(slug: $slug) {
+      id
+      hostMetrics {
+        hostFees {
+          value
+          currency
+        }
+        platformFees {
+          value
+          currency
+        }
+        pendingPlatformFees {
+          value
+          currency
+        }
+        platformTips {
+          value
+          currency
+        }
+        pendingPlatformTips {
+          value
+          currency
+        }
+        pendingHostFeeShare {
+          value
+          currency
+        }
+        totalMoneyManaged {
+          value
+          currency
+        }
+      }
+    }
+  }
+`;
+
+const Messages = defineMessages({
+  heading: {
+    id: 'TotalMoneyManagedSection.heading',
+    defaultMessage: 'Total Money Managed',
+  },
+  subHeading: {
+    id: 'TotalMoneyManagedSection.subHeading',
+    defaultMessage: 'My Organization and My Collectives',
+  },
+  totalAmountHeld: {
+    id: 'TotalMoneyManagedSection.totalAmountHeld',
+    defaultMessage: 'Total amount held in your bank account for the Host and its Collectives',
+  },
+  titleDescription: {
+    id: 'TotalMoneyManagedSection.titleTooltip',
+    defaultMessage: 'Total amount held in your bank account for the Host and its Collectives.',
+  },
+});
+
+const FundAmounts = styled.div`
+  width: 100%;
+  height: 48px;
+  border-radius: 10px;
+  background-color: #9a7bf1;
+  border-right: 350px solid #46347f;
+  padding-top: 10px;
+  padding-left: 5px;
+
+  @media (max-width: 1025px) {
+    height: 130px;
+    background-color: #9a7bf1;
+    border-right: 0px;
+    border-bottom: 40px solid #46347f;
+  }
+`;
+
+const TotalFundsLabel = styled(Container)`
+  display: table-cell;
+  padding-left: 10px;
+  height: 26px;
+  border-radius: 5px;
+  background-color: white;
+  opacity: 80%;
+  vertical-align: middle;
+`;
+
+const TotalMoneyManagedSection = ({ currency, hostSlug }) => {
+  const intl = useIntl();
+  const { loading, data } = useQuery(metricsQuery, {
```

Comment:
Since this data is always displayed, we should make it part of https://github.com/opencollective/opencollective-frontend/blob/enhancement/legal-name-placeholder/components/host-dashboard/HostDashboardReports.js#L19

Reply:
@Betree : Yes, that makes sense. I've moved it to the parent component. 👍🏽